lasagne/nonlinearities.py

```diff
@@ -78,7 +142,7 @@ def rectify(x):
     # and it works with nonsymbolic inputs as well.
     # Thanks to @SnipyHollow for pointing this out. Also see:
     # https://github.com/Lasagne/Lasagne/pull/163#issuecomment-81765117
-    return 0.5 * (x + abs(x))
+    return theano.tensor.nnet.relu(x)
```

Comment:
perhaps the comments above should be removed as well, since they refer to the particular implementation we were using.